Backend/lib/payment/spec/Storage/Wallet.yaml

```
@@ -19,11 +19,17 @@ PersonWallet:
     createdAt: UTCTime
     updatedAt: UTCTime
 
+  constraints:
+    personId: SecondaryKey
+
   queries:
     findByPersonId:
       kvFunction: findOneWithKV
       where: personId
 
+  constraints:
```

YAMLlint (1.38.0)

[error] duplication of key "constraints" in mapping

(key-duplicates)